index.js

```diff
@@ -4,7 +4,35 @@
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
  * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
  */
-function calculator() {}
+function calculator() {
+    let result = 0
+    let command = process.argv[2]
+
+      if (!command){
+         return "No operation provided...";
+    } else if (!process.argv[3]){
```

Comment:
style comment: try to use consistent indentation rules: this will make your code easier to read and debug!